habinezadalvan/Epic-Mail
# #164682487 Sets up the database

**what does this PR do?**
Sets up the database and links it to postgress.
**Description of Task to be completed**
-install postgress and other required packages
-create user/password/DB
-connect the database to psql
-create queries
**How should this be tested manually**
-clone repo
-install dependencies with npm install
-Run db with npm run db
-go in pgAdmin and see the created tables
**relative pivotal tracker story**
 #164682487
